config/fcos/v1_5_exp/translate.go

```diff
@@ -68,17 +68,31 @@ func (c Config) ToIgn3_4Unvalidated(options common.TranslateOptions) (types.Conf
 		return types.Config{}, translate.TranslationSet{}, r
 	}
 	r.Merge(c.processBootDevice(&ret, &ts, options))
+
 	for i, disk := range ret.Storage.Disks {
-		// In the boot_device.mirror case, nothing specifies partition numbers
-		// so match existing partitions only when `wipeTable` is false
-		if !util.IsTrue(disk.WipeTable) {
-			for j, partition := range disk.Partitions {
-				// check for reserved partlabels
-				if partition.Label != nil {
+		for p, partition := range disk.Partitions {
+			if partition.Label != nil {
+				if *partition.Label == "root" {
+					if partition.SizeMiB != nil && *partition.SizeMiB < 8192 {
+						r.AddOnWarn(path.New("json", "storage", "disks", i, "partitions", p, "sizeMiB", *partition.SizeMiB), common.ErrRootTooSmall)
+					} else {
+						for _, op := range disk.Partitions {
+							if op.Number > partition.Number {
+								if op.StartMiB == nil {
```

Comment:
This doesn't match the logic described in https://github.com/coreos/butane/issues/211#issuecomment-1205967355.